force-app/main/default/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -51,13 +51,19 @@ class psElevateTokenHandler {
         const url = `https://${domainInfo.orgDomain}--${namespace}.visualforce.com`;
         const alternateUrl = `https://${domainInfo.orgDomain}--${namespace}.${domainInfo.podName}.visual.force.com`;
         const productionEnhancedUrl = `https://${domainInfo.orgDomain}--${namespace}.vf.force.com`;
+        const productionEnhancedUrlLogin = `https://${domainInfo.orgDomain}--${namespace}.my.salesforce.com`;
         const sandboxEnhancedUrl =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.vf.force.com`;
```

Comment:
@voduyemi so oddly enough, as we discussed, these URLs technically do reflect the updates presented in the enhanced domains document https://help.salesforce.com/s/articleView?id=sf.domain_name_url_format_changes_enable_enhanced.htm&type=5. The one thing I am unsure of is if the `namespace` variable is properly reflecting the sandbox packagename when it's a sandbox environment. The sandbox url format is : `MyDomainName--SandboxName--PackageName.sandbox.vf.force.com`. I added some additional urls that could be good to have just in case as recognized origin urls. I also confirmed that this is the location where the bug is being triggered because I removed the `productionEnhancedUrl` from my dev code and triggered the bug in my scratch org, so I think the issue is now related to that sandbox url. Let me know if you'd like to discuss this as there are definitely some funky aspects to this!